### DISCUSSION TITLE
recent quasar versions changed styling significantly?

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#12216] [General - Components / Directives / etc] [answered]
So in one of the many recent quasar updates (I am using v1 branch, currently at 1.17.4) it seems that significant changes have been made to styling of inputs, and I am not sure whether this is a bug or I now need to adjust my settings. But for example I went from this a few weeks ago:

<img width="557" alt="Screen Shot 2022-01-27 at 9 59 34 AM" src="https://user-images.githubusercontent.com/288542/151326581-c9272623-4af6-4641-8f00-bd74ff60865a.png">
to this after recent updates:
<img width="563" alt="Screen Shot 2022-01-27 at 10 00 08 AM" src="https://user-images.githubusercontent.com/288542/151326646-b2957715-cd07-4449-86ad-6e66ee47d781.png">

So not super happy about it, since I don't see anything related to these elements that changed on my end, this happened after the upgrade. I have looked at the release notes, but it isn't obvious to me what exactly has changed and where the mitigations are/should be. Anyone else having issues like this?

**Comment by pdanpdan** [comment, marked as answer]
The QRange/QSlider change was intended (the line was made thicker by default, but now you can style it as you want)
For the center one I cannot see any difference.
For the left one - I need a codepen to see what is there, because I don't remember of any change related to that.

**Reply by ssuess** [reply]
Thanks for the repsonse, but I can't find the parameter that sets the line thickness here: https://v1.quasar.dev/vue-components/slider#introduction

As for the other two, they both now mysteriously have this padding at the bottom (whether activated as on left, or not as in center)

**Reply by ssuess** [reply]
oh wait (for the slider problem) it is track-size thanks. (just seemed odd the default value was 35 so I missed it)

**Reply by pdanpdan** [reply]
Track size :) - It's not the default, it's an example
https://codepen.io/pdanpdan/pen/YzEPOgW?editors=1000

About the padding - check if the QRange is not stretching the container (remove QRange and see if you still have padding)
Then put QRange back and determine the height with the inspector

**Reply by ssuess** [reply]
That was it after all, thanks. I had to add an explicit height to the slider div, and that fixed it. Thanks again for your help.

**Reply by pdanpdan** [reply]
You can also use `dense` prop for QSlider/QRange
